### PR TITLE
Change slack webhook

### PR DESCRIPTION
**Pull request by Birajpjpt**
## Change

Change the slack webhook from Squirrelfarm to Peakbot

## Problem / Why

Squirrelfarm is now retired, so makes sense to use Peakbot until Chimera is ready to be used. 

## References

- https://workday-dev.slack.com/archives/GBK5KEHAP/p1706605080974119

### CODE REVIEW

File: scripts/git_api_curl.sh [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 
 # curl -X POST -H "Accept: application/json" -H "Authorization: Bearer $GITHUB_SECURITY_ACCESS_TOKEN" https://api.github.com/repos/$BUILDKITE_ORGANIZATION_SLUG/$BUILDKITE_PIPELINE_SLUG/commits/$BUILDKITE_COMMIT/comments -d '{"body":"A new potential secret was discovered in this commit @peakon/security"}'
-curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"Potential secret in Github!\", \"attachments\":[{\"color\": \"warning\", \"title\": \"A new potential secret was commited in Github\", \"text\": \"https://github.com/$BUILDKITE_ORGANIZATION_SLUG/$BUILDKITE_PIPELINE_SLUG/commit/$BUILDKITE_COMMIT\"}]}" $SF_DEFAULT_SLACK_WEBHOOK_URL
-
+curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"Potential secret in Github!\", \"attachments\":[{\"color\": \"warning\", \"title\": \"A new potential secret was commited in Github\", \"text\": \"https://github.com/$BUILDKITE_ORGANIZATION_SLUG/$BUILDKITE_PIPELINE_SLUG/commit/$BUILDKITE_COMMIT\"}]}" $PEAKBOT_WEBHOOK_SECRET
```
Comment (threesquared) [inline scripts/git_api_curl.sh]: I am not sure this is the right secret, this is for authenticating to peakbot

**Review by vgrigoruk — APPROVED** [review]

File: scripts/git_api_curl.sh (hunk 1 above)
Comment (threesquared) [reply scripts/git_api_curl.sh]: You either need to use [this](https://github.com/peakon/ci-scripts-buildkite-plugin/blob/6450f700687c81bd7a00b1745cda91312539ad98/bin/notify-slack-channel) method to send via the peakbot API or you can get the `peakbot-slack-token` secret and use it to call the slack API [directly](https://api.slack.com/tutorials/tracks/posting-messages-with-curl)

Comment (Birajpjpt) [reply scripts/git_api_curl.sh]: @threesquared Have we stored `peakbot-slack-token` in param store? I can't find it in CI.

Comment (threesquared) [reply scripts/git_api_curl.sh]: Ah I think maybe only ArgoCD uses it, can you copy it from the management account?

Comment (Birajpjpt) [reply scripts/git_api_curl.sh]: I don't have access to add to param store

Comment (threesquared) [reply scripts/git_api_curl.sh]: Added as `/vendors/buildkite/deploy/global/env/PEAKBOT_SLACK_TOKEN` now

Comment (Birajpjpt) [reply scripts/git_api_curl.sh]: Recent changes in [3d0bcda](https://github.com/peakon/docker-detect-secrets/pull/9/commits/3d0bcda4e1de6fbc8abcdff0b6a31d3de989df07).

**Review by threesquared — APPROVED** [review]
LGTM

**Pull request merged**